content/docs/user-guide/programming/debugging/tracing.md

```diff
@@ -13,14 +13,17 @@ The following table describes the `AZ_Tracing` macros and their uses.
 
 ****
 
-| AZ Tracing Macro | Description |
-| --- | --- |
-| `AZ_Assert` |  Use for critical errors when the program cannot continue. `AZ_Assert` macros print an error message, the file name and line number where the error occurred, and then break program execution.  |
-| `AZ_Error` |  Use when an obvious error has occurred but the program can continue safely. `AZ_Error` macros print an error message and the file name and line number where the error occurred. In some environments, `AZ_Error` notifies the user that an error has occurred.  |
-| `AZ_Warning` |  Use when an error might have occurred. `AZ_Warning` macros print an error message and the file name and line number of the possible error, but take no other action.  |
-| `AZ_TracePrintf` |  Use for informational purposes only. `AZ_TracePrintf` macros print a message but take no other action.    |
-
- For source code, see `Code\Framework\AzCore\AzCore\Debug\Trace.*`.
+| AZ Tracing Macro | Description                                                                                                                                                                                                                                                     |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `AZ_Assert`      | Use for critical errors when the program cannot continue. `AZ_Assert` macros print an error message, the file name and line number where the error occurred, and then break program execution.                                                                  |
```

Comment:
I'm wondering if this is a good place to mention the existence and behavior of the `sys_asserts` cvar that configures whether AZ_Asset is disabled, logging-only, UI prompt, or, crash. Or whether that is already explained somewhere that we could link to from here.